pandas/tests/indexes/datetimes/test_tools.py

```diff
@@ -409,8 +409,8 @@ def test_to_datetime_tz_pytz(self, cache):
                                                      hour=3, minute=0))],
                        dtype=object)
         result = pd.to_datetime(arr, utc=True, cache=cache)
-        expected = DatetimeIndex(['2000-01-01 08:00:00+00:00',
```

Comment:
I find this surprising that you change this

Reply:
Thanks, yeah this was the wrong fix. Addressed